packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -195,7 +181,7 @@ const SignInVerificationMachine = setup({
     formRef: input.formRef,
     loadingStep: 'verifications',
     parent: input.parent,
-    registeredStrategies: new Set<SignInFactor>(),
+    registeredStrategies: new Set<SignInStrategyName>(),
```

Comment:
This change stems from looking at `packages/elements/src/react/sign-in/verifications.tsx` and checking what is actually passed